File: Tutorial/Onboarding/Python/Onboarding.py
import sys

while 1:
    enemy_1_name        = raw_input()  
    enemy_1_distance    = int(raw_input())  
    enemy_2_name        = raw_input()  
    enemy_2_distance    = int(raw_input()) 
    
    print (enemy_1_name if enemy_1_distance < enemy_2_distance else enemy_2_name)

File: Tutorial/Onboarding/Ruby/Onboarding.rb
STDOUT.sync = true

loop do
  $enemy_1_name = gets.chomp
  $enemy_1_distance = gets.to_i
  $enemy_2_name = gets.chomp
  $enemy_2_distance = gets.to_i

  puts (if $enemy_1_distance < $enemy_2_distance then $enemy_1_name else $enemy_2_name end)
end